<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextFieldSettings</class>
 <widget class="QWidget" name="TextFieldSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" name="layOptions">
     <item row="1" column="0">
      <widget class="QCheckBox" name="chkDisabled">
       <property name="text">
        <string>Disabled</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblPlaceholder">
       <property name="text">
        <string>Placeholder</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblText">
       <property name="text">
        <string>Text</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="chkInputLine">
       <property name="text">
        <string>Input line</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="chkLabel">
       <property name="text">
        <string>Label</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="txtText">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QLineEdit" name="txtPlaceholder">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblLabel">
       <property name="text">
        <string>Label</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="txtLabel"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="MaterialTextField" name="textField"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaterialTextField</class>
   <extends>QLineEdit</extends>
   <header location="global">QtMaterialWidgets/textfield.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
